incubator-tools/language_detection/language_detection.ipynb

```diff
@@ -0,0 +1,535 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.        document_input_config = {
</pre><p>These dicts should be using the Classes to ensure type safety</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/700/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Updated. Replaced by the folowing lines of code : <code>&nbsp;gcs_source = translate.GcsSource(input_uri=gcs_input_path)</p><p>&nbsp;&nbsp;document_input_config = translate.DocumentInputConfig(gcs_source=gcs_source)</p><p>&nbsp;&nbsp;gcs_destination = translate.GcsDestination(output_uri_prefix=gcs_output_path)</p><p>&nbsp;&nbsp;document_output_config = translate.DocumentOutputConfig(gcs_destination=gcs_destination)</p><p>&nbsp;&nbsp;&nbsp;</p><p>&nbsp;&nbsp;request = translate.TranslateDocumentRequest(</p><p>&nbsp;&nbsp;&nbsp;&nbsp;parent=parent,&nbsp;</p><p>&nbsp;&nbsp;&nbsp;&nbsp;target_language_code='en',</p><p>&nbsp;&nbsp;&nbsp;&nbsp;document_input_config=document_input_config,</p><p>&nbsp;&nbsp;&nbsp;&nbsp;document_output_config=document_output_config</p><p>&nbsp;&nbsp;)</p><p>&nbsp;&nbsp;response = client.translate_document(request)</code></p>